practice-app/api/urls.py

```diff
@@ -1,7 +1,6 @@
-from django.urls import path, include
+from django.urls import path
 from . import views
 
-
-
 urlpatterns = [
+    path('posts/covid/<int:post_id>', views.get_covid_numbers, name='covid_numbers'),
```

Comment:
I think parameter here should be story_id. I could not run the tests with post_id.

Reply:
Thank you for spotting the mistake here, fixing it immediately!